src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1396,6 +1429,109 @@ protected override void StopProcessing()
 
         #region Helper Methods
 
+        
+        /// <summary>
+        /// Adds content to a <see cref="MultipartFormDataContent" />. Object type detection is used to determine if the value is String, File, or Collection.
+        /// </summary>
+        /// <param name="fieldName">The Field Name to use.</param>
+        /// <param name="fieldValue">The Field Value to use.</param>
+        /// <param name="formData">The <see cref="MultipartFormDataContent" />> to update.</param>
+        /// <param name="enumerate">If true, collection types in <paramref name="fieldValue" /> will be enumerated. If false, collections will be treated as single value.</param>
+        internal void AddMultipartContent (object fieldName, object fieldValue, MultipartFormDataContent formData, bool enumerate)
```

Comment:
Extra space after method name - we never use the pattern. Below too.

Shouldn't the method be private? Helper methods below too.

Reply:
I'm not sure. I was following suit with the existing helper functions. I will try setting them private and see if anything breaks.

Reply:
We used `internal` because we had many files.

Reply:
fixed